invokeai/frontend/web/src/features/parameters/store/generationSlice.ts

```diff
@@ -26,6 +27,9 @@ import {
 } from '../types/parameterSchemas';
 
 export interface GenerationState {
+  hrfScale: HrfScaleParam;
+  hrfToggled: boolean;
```

Comment:
I'd prefer this to be `hrfIsEnabled` for clarity, because `hrfToggled` only indicates that this is a boolean, not what `true` means

Reply:
good point, how about `hrfEnabled` since the 'Is' is implied?

Reply:
Sure both are clear